docs/javascript/v2/plugins/virtual-background.mdx

```diff
@@ -42,6 +42,7 @@ This accepts background as a parameter, It can be of 3 types:
 -  `blur` - String - This will set the background to blur
 -  `image` - HTMLImageElement - This will replace the background to the image provide
 -  `none` - String - This will remove the background effect from the video
+-  `gifURL` - String - This is the URL of the gif with which the background will be replaces
```

Comment:
replaces => replaced

Video also needs to be added right?

Reply:
done